providers/PaperProvider/src/main/java/net/ess3/provider/providers/PaperAsyncChatListenerProvider.java

```diff
@@ -32,12 +35,17 @@ public PaperAsyncChatListenerProvider() {
 
     @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
     public void onAsyncChatEvent(final AsyncChatEvent event) {
-        Bukkit.getPluginManager().callEvent(new AbstractAsyncChatEvent(
-                event.getPlayer(),
-                serializer.serialize(event.message()),
-                event.viewers().stream()
-                        .filter(v -> v instanceof Player)
-                        .map(v -> (Player) v)
-                        .collect(Collectors.toSet())));
+        Bukkit.getScheduler().runTaskAsynchronously(plugin, () ->
```

Comment:
use ``ess.runTaskAsynchronously(() -> {``

Reply:
I found a workaround so everything works without having to run a task. Thanks for your advice!